### PR TITLE
Merging issue/7/Support-duplicated-country-names into master

### PR DESCRIPTION
**Pull request by inakiesteve**
Implementing issue #7 in repository.

This includes: 

- Adding control to avoid crash when same country name is exported more than once.
- Changing airport and country provider url.
- Fixing some tests that were failing due to not handling duplicated country names properly.
- Adding more tests to guarantee airports and countries are imported properly.

### CODE REVIEW

File: Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData.Tests/OpenFlightsDataProviderTests.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@
 using Ibistic.Public.OpenAirportData.OpenFlightsData;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-namespace Ibistic.Public.OpenAirportData.Tests
+namespace Ibistic.Public.OpenAirportData.Tests2
```
Comment (magnusakselvoll) [inline Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData.Tests/OpenFlightsDataProviderTests.cs]: I don't see a reason to change this namespace.

File: Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData.Tests/OpenFlightsDataProviderTests.cs [hunk 2]
```diff
@@ -216,13 +216,21 @@ public void TestValidCountries()
         [TestMethod]
         public void TestListInvalidAirports()
         {
-            Dictionary<string, Country> countries;
+            Dictionary<string, Country> countries = new Dictionary<string, Country>();
```
Comment (magnusakselvoll) [inline Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData.Tests/OpenFlightsDataProviderTests.cs]: var

File: Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData/OpenFlightsData/OpenFlightsDataAirportProvider.cs [hunk 3]
```diff
@@ -50,15 +50,25 @@ public IEnumerable<Airport> GetAllAirports()
...
             var airports =  _csvReader.GetRecords<Airport>();
 
             return _countryProvider == null ? airports : AddCountryInformation(airports);
         }
 
         private IEnumerable<Airport> AddCountryInformation(IEnumerable<Airport> airports)
         {
-            Dictionary<string, Country> countriesByName = _countryProvider.GetAllCountries().Where(x => !String.IsNullOrEmpty(x.Name))
-                .ToDictionary(x => x.Name);
+            Dictionary<string, Country> countriesByName = new Dictionary<string, Country>();
```
Comment (magnusakselvoll) [inline Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData/OpenFlightsData/OpenFlightsDataAirportProvider.cs]: var

File: Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData.Tests/OpenFlightsDataProviderTests.cs (hunk 1 above)
Comment (inakiesteve) [reply Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData.Tests/OpenFlightsDataProviderTests.cs]: should be ok now

File: Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData.Tests/OpenFlightsDataProviderTests.cs (hunk 2 above)
Comment (inakiesteve) [reply Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData.Tests/OpenFlightsDataProviderTests.cs]: done

File: Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData/OpenFlightsData/OpenFlightsDataAirportProvider.cs (hunk 3 above)
Comment (inakiesteve) [reply Ibistic.Public.OpenAirportData/Ibistic.Public.OpenAirportData/OpenFlightsData/OpenFlightsDataAirportProvider.cs]: done

**Comment by magnusakselvoll** [comment]
You need to change the version in the project file so that we can publish a new nuget component. I suggest setting it to 1.1.

Also, add the changes to the release notes (PackageReleaseNotes), e.g. 
1.1.0:
   - Accepting duplicates in source data
   - Pinning source data to specific version
   - Upgraded dependency to CsvHelper to version 12.2.1

**Pull request merged**